README.md

```diff
@@ -212,6 +209,6 @@ I would like to thank [Microverse](https://github.com/microverseinc) for providi
 
 ## 📝 License <a name="license"></a>
 
-This project is [MIT](./MIT.md) licensed.
+This project is [GNU](./LICENSE.md) licensed.
```

Comment:
- Hey, the licensed file is still broken as pointed out by the previous reviewer, please, ensure you fix this correctly to make the readme more professional.